swagger.yaml

```diff
@@ -686,10 +686,32 @@ paths:
             type: integer
         - name: productId
```

Comment:
Id say that this filter could be removed. Originally the logic in the UI was:

- Listing all products
- Search filter was actually based on product code
- User would type the product code
- UI would search the product from products list based on that given code
- If product was found UI would send a listInstructions request with productId filter

Now the idea is to the filtering to the API side. So, if productCode filter is given the user usually knows the specific code that he/she is searching for (strict filter).

If productName is given then user is trying to find something based on partial name match from the dropdown (which should be autocomplete field that sends a listInstructions call when the name search value is changed).

Reply:
okay, so only 2 filters. atm if both strict and not-strict filters are used, they are applied together